src/bun.js/bindings/exports.zig

```diff
@@ -1973,6 +1965,15 @@ pub const ZigConsoleClient = struct {
             };
         }
 
+        fn getObjectName(globalThis: *JSC.JSGlobalObject, value: JSValue) ZigString {
+            var name_str = ZigString.init("");
+            value.getClassName(globalThis, &name_str);
+            if (name_str.len > 0 and !strings.eqlComptime(name_str.slice(), "Object")) {
```

Comment:
@paperdave this code has a mistake in it (which was likely copied from elsewhere)

It's asserting that the class name is always ascii.  It could be UTF-16. 

Instead, we can do `.eqlComptime` on the string itself.

Reply:
My bad. I copied this from https://github.com/oven-sh/bun/blob/68324daf7852343d895e415062f5ad659b511bee/src/bun.js/bindings/exports.zig#L1828.

I didn't replicate your update because I assumed incorrectly that it was the difference between ZigString vs BunString